src/HealthChecks.Dapr/HealthChecks.Dapr.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
I wasn't sure how to approach the `TargetFrameworks` here, because `Dapr.Client` targets only **.NET Core 3.1**, so I couldn't target **.NET Standard 2.0** in the package.

My idea was to target only supported .NET versions for now.

I can also add .NET Core 3.1 (EOL December '22) and .NET 5.0 (EOL May 2022), if we want to make it backwards available to folks using the older versions of .NET.

Please advise @sungam3r 😄

Reply:
Targeting 3.1 in libraries is fine. You may target all 3 tfm but be careful with dependencies, see Condition attribute in csprojs.

Reply:
I'll merge as is.